### PR TITLE
[ctpg] Fix portfile

### PR DESCRIPTION
**Pull request by Som1Lse**
- #### What does your PR fix?
  Removes three lines from `ports/ctpg/portfile.cmake`, that copy the `share` from the `debug` directory into the main directory. `ctpg` is header-only so the files aren't needed, and they end up using an invalid include path, which made the library unusable. (Tested on `x64-windows`.)

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
  Same as before, everything. No.

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
  Yes.

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
  Yes.

### CODE REVIEW

**Comment by Som1Lse** [comment]
Huh, that is really interesting. I get the same thing. Any idea why it does that?

I'm gonna inspect the generated files a bit more closely, sec.

**Comment by Som1Lse** [comment]
Okay, found the issue:
1. `ctpg-config.cmake` is installed into `share/ctpg/cmake`.
2. `vcpkg` finds `ctpg-config.cmake` and looks at the parent directory, called `cmake`, and prints that.

Solution: Use `vcpkg_cmake_config_fixup(CONFIG_PATH share/${PORT}/cmake)` and let `vcpkg` do the hard work.

I'll be pushing this, bumping `port-version` again.

**Comment by dg0yt** [comment]
> Huh, that is really interesting. I get the same thing. Any idea why it does that?

#20190

**Comment by Som1Lse** [comment]
Yeah, I figured it out. Though now the CI seems to be stuck.

**Comment by Som1Lse** [comment]
> After cloning your repository, install `ctpg` and find some problems with `usage`, find_package(cmake ... :
> 
> ```
> ctpg provides CMake targets:
> 
>     # this is heuristically generated, and may not be correct
>     find_package(cmake CONFIG REQUIRED)
>     target_link_libraries(main PRIVATE ctpg::ctpg)
> ```

This should now be fixed. I get following usage information:
```
ctpg provides CMake targets:

    # this is heuristically generated, and may not be correct
    find_package(ctpg CONFIG REQUIRED)
    target_link_libraries(main PRIVATE ctpg::ctpg)
```
As far as I can tell, that look correct.

File: versions/c-/ctpg.json [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
+    {
+      "git-tree": "e9e422634e91620aeb44013c59d7c0e7731f3ba4",
+      "version": "1.3.7",
+      "port-version": 3
+    },
+    {
+      "git-tree": "59a666ff9f67ea79c9eca77d981b2ceecad89d0a",
+      "version": "1.3.7",
+      "port-version": 2
+    },
```
Comment (MonicaLiu0311) [inline versions/c-/ctpg.json:12]: Each PR can only generate one record at most, please delete these two records and execute the following command:
* Modify the `port-version` of the `vcpkg.json` file to `2`
* Run `./vcpkg x-add-version ctpg` to regenerate a record
* commit+push

PS: If you modify and push multiple times, use the command `./vcpkg x-add-version ctpg` for the first time, and use `./vcpkg x-add-version ctpg --overwrite-version` multiple times later

Comment (Som1Lse) [reply versions/c-/ctpg.json]: Done.

Maybe a mention of `--overwrite-version` should be added to the maintainer guide? I couldn't find a mention of it anywhere in the docs. Then again, it probably only comes up rarely.

Comment (MonicaLiu0311) [reply versions/c-/ctpg.json]: Thanks for your feedback.

**Review by MonicaLiu0311 — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for the fix!

**Pull request merged**